api/lib/domain/read-models/CertificationReferent.js

```diff
@@ -0,0 +1,25 @@
+class CertificationReferent {
```

Comment:
Attention, je ne sais pas si le terme est déjà utilisé ailleurs dans le code mais "Referent" en anglais n'inclut pas le sens qu'on lui donne ici de "personne responsable". Au mieux "certification referent" pourrait être compris comme "ce à quoi fait référence une certification"…

Reply:
Referrer ? @jonathanperret

Reply:
Damn ce que c'est dur de trouver des mots dans sa langue pas d'origine (i see you `prescriber`)

Reply:
Non, `Referrer` c'est celui qui te réfère, c'est encore trop "collé" à un sens de "référent" qui est assez particulier en français… qui d'ailleurs semble avoir été importé de l'anglais 
 de la médecine ("Referring physician") récemment puis déformé, d'après http://jeanreneloubat.fr/pdf/Archives_dernieres-reflexions/de-la-reference-a-la-coordination.pdf

Il faut revenir au concept métier plutôt que d'essayer de traduire littéralement. En l'occurrence on parle d'un utilisateur de Pix Certif, c'est quelqu'un qui va… organiser des certifications ? Les surveiller ? Les administrer ?

Reply:
Je veux éviter le piège évident de Manager, c'est tellement fourre-tout... Mais ok j'y réfléchis

Reply:
Leader ?

Reply:
CertificationPointOfContact